site/src/docs/components/koros/accessibility.mdx

```diff
@@ -10,5 +10,7 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 ## Accessibility
 
 ### Pay attention to
-- HDS Footer is designed to support many different brand colours. When customising colours, refer to <InternalLink href="/foundation/design-tokens/colour">colour guidelines</InternalLink> to ensure accessibility.
-- **When designing the footer link hierarchy, always think about keyboard and screen reader users.** Remember that keyboard users need to manually navigate to each element.
+
```

Comment:
These three are not really accesibility concerns, but should be included in the general principles